.devops/templates/build-test-lint.yml

```diff
@@ -44,7 +44,11 @@ steps:
       DANGER_GITHUB_API_TOKEN: $(DANGER_GITHUB_API_TOKEN)
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31533&commitId=a74ae978bd54f3fccb21c1f21b0e76a7610cc0d7&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|4| Changed|
|Avatar Converged.customSize+image.normal.chromium.png|0| Added|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Variations Fallback.normal.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default - High Contrast.default.chromium.png|0| Removed|
|SwatchPicker Converged.spacing.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>